R/readThresholdFile.R

```diff
@@ -15,20 +15,23 @@
 # limitations under the License.
 
 .readThresholdFile <- function(checkThresholdLoc, defaultLoc) {
+  thresholdFile <- checkThresholdLoc
   if (checkThresholdLoc == "default") {
-    result <- read_csv(
-      file = system.file(
-        "csv",
-        defaultLoc,
-        package = "DataQualityDashboard"
-      ),
-      na = c(" ", "")
-    )
-  } else {
-    result <- read_csv(
-      file = checkThresholdLoc,
-      na = c(" ", "")
+    thresholdFile <- system.file(
+      "csv",
+      defaultLoc,
+      package = "DataQualityDashboard"
     )
   }
+  colspec <- readr::spec_csv(thresholdFile)
+  # plausibleUnitConceptIds is a comma-separated list of concept ids, but it is being interpreted as col_double()
+  if ('plausibleUnitConceptIds' %in% names(colspec$cols)) {
+    colspec$cols$plausibleUnitConceptIds <- readr::col_character()
+  }
+  result <- read_csv(
+    file = thresholdFile,
+    col_types = colspec,
+    na = c(" ", "")
+  )
```

Comment:
for some reason the diff here isn't showing [this](https://github.com/OHDSI/DataQualityDashboard/blob/eb8a749b6a6ffeeca1c8c2ab68f62a90d03835f6/R/readThresholdFile.R#L40) change i added in v2.2. can you try rebasing again off the develop branch? i think doing that should pull in the right state of develop here.